parser/parser.go

```diff
@@ -51,6 +51,8 @@ func New(l *lexer.Lexer) *Parser {
 
 	p.prefixParseFns = make(map[token.Type]prefixParseFn)
 	p.registerPrefix(token.Ident, p.parseIdentifier)
+	p.registerPrefix(token.Class, p.parseIdentifier)
```

Comment:
I think this is unnecessary? Because if `class` or `module` are  used as method call, they should be classified as  `Ident` when tokenizing.